processor/resourcedetectionprocessor/internal/resourcedetection.go

```diff
@@ -128,12 +128,11 @@ func (p *ResourceProvider) detectResource(ctx context.Context) {
 	for _, detector := range p.detectors {
 		r, schemaURL, err := detector.Detect(ctx)
 		if err != nil {
-			p.detectedResource.err = err
-			return
+			p.logger.Debug("failed to detect resource", zap.Error(err))
```

Comment:
I wonder if this should be warning since the knock-on effects are potentially substantial and debug logging isn't commonly used in a standard deployment.

Reply:
Thanks, @rmfitzpatrick for drawing attention, We have changed it to a warning.